### PR TITLE
Runtime attribute class resolution

### PR DESCRIPTION
**Pull request by eikes**

### CODE REVIEW

File: spec/ioki/model/models_without_open_api_specifications_spec.rb [hunk 1]
```diff
@@ -0,0 +1,213 @@
+# frozen_string_literal: true
+
+RSpec.describe 'models without open api specifications', vcr: { record: :all } do
```
Comment (A5308Y) [inline spec/ioki/model/models_without_open_api_specifications_spec.rb:3]: I split this file up and their shouldn't be `record: :all` on CI. Let me have a look in master.

Comment (A5308Y) [reply spec/ioki/model/models_without_open_api_specifications_spec.rb:3]: Yes. Could you start from/rebase on master?

Comment (A5308Y) [reply spec/ioki/model/models_without_open_api_specifications_spec.rb:3]: Actually deleting the file and the vcr cassettes should do the trick.

File: lib/ioki/model/base.rb [hunk 2]
```diff
@@ -166,6 +167,18 @@ def reset_attributes!
         end
       end
 
+      # Tries to find a constant in the current modules parent module with the specified name
+      #
+      # class X::Y
+      #   constantize_in_module('Z') # => X::Z
```
Comment (ans-ioki) [inline lib/ioki/model/base.rb:173]: i'd opt to remove this. I think one can read the code straight forward. you can put it into the PR description. i'd rather not have inline doc.

File: spec/ioki/model/models_without_open_api_specifications_spec.rb (hunk 1 above)
Comment (eikes) [reply spec/ioki/model/models_without_open_api_specifications_spec.rb:3]: Yes. I don't know why the WIP commits are in this PR at all.

**Review by A5308Y — APPROVED** [review]
LGTM!

**Comment by eikes** [comment]
Should I go ahead and merge this or are we waiting for someone or something?

**Comment by A5308Y** [comment]
@eikes I did already leave for the holidays. You can merge your PRs if you have at least one approval.

**Pull request merged**